ansible/inventory.yml

```diff
@@ -78,6 +78,9 @@ hosts:
         win2012r2-x64-2: {ip: 104.209.212.27, port: 61718}
         win2012r2-x64-3: {ip: 104.209.210.218, port: 52109}
 
+    - cloudapp:
+        ubuntu1404-x64-1: {ip: node-msft-cross-compiler-1.cloudapp.net}
```

Comment:
The provider should be `azure_msft`, so something like:

```
- azure:
    msft-ubuntu1404-x64-1: ...
```

The same is true for the Windows servers above, but they were added as a draft by Johan and have not been reviewed yet.